frontend/apps/reader/modules/readertypeset.lua

```diff
@@ -136,16 +138,19 @@ end
 
 function ReaderTypeset:onToggleEmbeddedStyleSheet(toggle)
     self:toggleEmbeddedStyleSheet(toggle)
+    Notification:notify(T( _("Embedded styles are %1."), optionsutil:getOptionText("ToggleEmbeddedStyleSheet", toggle)))
```

Comment:
Please don't do this, but use full sentences instead.

```
if toggle then
 Embedded styles are enabled.
else
 Embedded styles are disabled.
```

The fact that this happens to work in English is coincidental.

Reply:
No problem to change. But I didn't get the point. 
Do you mean that "off" translates to something strange in this sentence in some languages?
Would we have to change the messages for "Font kerning" too?

Reply:
If font kerning doesn't use a numerical value, ideally, yes. But a simple workaround is to quote or use a colon. (i.e., `set to: some value` or `set to 'some value'`). It's probably not worth the effort to do better.

You don't have to go far to see how concatenating strings like a programmer could be problematic.

In French, for example, you might get something like:

* Font kerning (masculin) is désactivé.
* Embedded styles (feminine) is désactivée.

NB The genders above are completely fictional. Both style and kerning are masculin in French. ;-)

When reasonably possible, always try to use full sentences no matter how redundant they might seem. Don't DRY. People outside of the Anglo-Dutch-German bubble will thank you. ;-D

Reply:
Thanks for the explanation.
As this feature will not be used much, and we don't want to double all the strings in `creoptions` I would go for `blabla set to: somevalue`